src/Core/Settings.h

```diff
@@ -1080,6 +1080,7 @@ class IColumn;
     M(String, format_schema, "", "Schema identifier (used by schema-based formats)", 0) \
     M(String, format_template_resultset, "", "Path to file which contains format string for result set (for Template format)", 0) \
     M(String, format_template_row, "", "Path to file which contains format string for rows (for Template format)", 0) \
+    M(String, format_schema_rows_template, "", "Format string for rows (for Template format)", 0) \
```

Comment:
Let's rename it to `format_template_row_schema` or `format_template_row_format`. We should keep `format_template_` prefix as it indicates that this setting is related only to format Template